styles/rating.css

```diff
@@ -0,0 +1,34 @@
+.rating.main-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 3rem;
+    width: 900px;
+    margin-left: 5rem;
+}
+
+.rating-container{
+    border: 1px solid white;
+    padding: 1rem 2rem;
+    border-radius: 8px;
+}
+
+.empty-rating, .filled-rating, .custom-rating, .emoji-rating{
```

Comment:
make a class of flex and use that in all the divs where it is required, much faster than this.